.github/workflows/renate-od.yml

```diff
@@ -0,0 +1,48 @@
+name: RENATE-OD CI
+
+on:
```

Comment:
Should run for every push, regardless of pull request, if possible.

Reply:
travis.yml should be removed.

Reply:
Both Pass and Fail should be tested.

Reply:
> Should run for every push, regardless of pull request, if possible.

It is possible:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push

PR is updated.

Reply:
> travis.yml should be removed.

Removed.